src/PIL/IptcImagePlugin.py

```diff
@@ -18,10 +18,9 @@
 import tempfile
 
 from . import Image, ImageFile
-from ._binary import i8
+from ._binary import i8, o8
```

Comment:
What happened here? Does Ruff not support isort's Black profile?

Reply:
isort is a bit looser than Ruff with these; Ruff wants either everything or nothing to be grouped like this, and isort doesn't always care.

Reply:
Testing, isort always sorts this like the original, with or without the Black profile.

Anyway, I don't mind too much, as long as it remains relatively stable.

On that topic, maybe we should wait until Ruff is at "v1" or "stable", which looks to be close:

https://github.com/charliermarsh/ruff/issues/1992

Reply:
Ruff has Black profile as the ONLY profile possible.